src/Adapters/Laravel/Console/Commands/PanDeleteCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pan\Adapters\Laravel\Console\Commands;
+
+use Illuminate\Console\Command;
+use Pan\Contracts\AnalyticsRepository;
+
+class PanDeleteCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'pan:delete {id : The ID of the analytic to delete}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Delete analytics by ID.';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(AnalyticsRepository $repository): void
```

Comment:
Thoughts on this?

public function handle(AnalyticsRepository $repository): void
{
    $id = (int) $this->argument('id');

    if ($this->isInvalidId($id)) {
        $this->error('Analytic ID must be greater than 0.');

        return;
    }

    if ($repository->delete($id)) {
        $this->info('Analytic record deleted.');
    } else {
        $this->error('Failed to delete the analytic record.');
    }
}

private function isInvalidId(int $id): bool
{
    return $id <= 0;
}

Reply:
Good one. I have added already.